R/transform.R

```diff
@@ -4,11 +4,17 @@
 #'   the result.
 #' @param files A character vector with paths to the file that should be
 #'   transformed.
+#' @param assert_code Boolean to indicate whether checking for parsing errors
+#'   should be done prior to styling.
 #' @inheritParams make_transformer
 #' @return A logical value that indicates whether or not any file was changed is
 #'   returned invisibly. If files were changed, the user is informed to
 #'   carefully inspect the changes via a message sent to the console.
-transform_files <- function(files, transformers) {
+transform_files <- function(files, transformers, assert_code = TRUE) {
```

Comment:
I'm not sure we need a new argument here. Rather the behavior should always be reasonable if parsing fails. If necessary, we could change `transform_lines_enc()` to account for this.